dot/parachain/candidate-validation/messages.go

```diff
@@ -48,3 +48,17 @@ const (
 	PreCheckOutcomeInvalid
 	PreCheckOutcomeFailed
 )
+
+const (
+	// Maximum compressed code size we support right now.
+	// At the moment we have runtime upgrade on chain, which restricts scalability severely. If we want
+	// to have bigger values, we should fix that first.
+	//
+	// Used for:
+	// * initial genesis for the Parachains configuration
+	// * checking updates to this stored runtime configuration do not exceed this limit
+	// * when detecting a code decompression bomb in the client
+	// NOTE: This value is used in the runtime so be careful when changing it.
+	MaxCodeSize             = 3 * 1024 * 1024
+	ValidationCodeBobmLimit = MaxCodeSize * 4
```

Comment:
what is BobmLimit here?

Reply:
BombLimit was the max size that a validation code blob could be, this was included in an earlier release in the polkadot repo, but this check has since been removed and is no longer in the current code base.  So, I'm going to remove this from ours.